### ISSUE TITLE
Vectorized netmask & hostmask

### ISSUE THREAD

**Issue by TomAugspurger** [ContinuumIO/cyberpandas#22]
For a single addr

```python
In [36]: import ipaddress

In [37]: addr = ipaddress.IPv4Address('192.168.1.1')

In [38]: net = ipaddress.IPv4Network(addr)

In [39]: net
Out[39]: IPv4Network('192.168.1.1/32')

In [40]: net.netmask
Out[40]: IPv4Address('255.255.255.255')

In [41]: net.hostmask
Out[41]: IPv4Address('0.0.0.0')
```

For an IPArray of values, we would accept a string `'192.168.1.1'` or the `IPv*Network` object.

**Comment by TomAugspurger** [comment]
API design question: From the stdlib `ipaddress` module, `.netmask` and `.hostmask` are available on network objects.

```python
In [6]: ip_network(ip_address('192.168.1.1')).netmask
Out[6]: IPv4Address('255.255.255.255')
```

AFAICT, the netmask / hostmask will be the same for every address, since it'll use the default `/32` when creating the network from the address.

```python
In [14]: ip_network('192.0.0.0/8').netmask
Out[14]: IPv4Address('255.0.0.0')
```

Do we want a function like?

```
def netmask(self, length=32) -> IPArray
    ...
```

**Comment by TomAugspurger** [comment]
cc @seibert for that ^

**Comment by TomAugspurger** [comment]
Something like

```python
In [1]: from cyberpandas import IPArray

In [2]: arr = IPArray(['192.0.0.0', '192.168.0.0', '192.168.1.0', '192.168.1.1',
   ...:                2**64 + 10])
   ...: arr
   ...:
Out[2]: IPArray(['192.0.0.0', '192.168.0.0', '192.168.1.0', '192.168.1.1', '::1:0:0:0:a'])

In [3]: arr.netmask()
Out[3]: IPArray(['255.255.255.255', '255.255.255.255', '255.255.255.255', '255.255.255.255', 'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff'])

In [4]: arr.netmask(v4_prefixlen=8, v6_prefixlen=64)
Out[4]: IPArray(['255.0.0.0', '255.0.0.0', '255.0.0.0', '255.0.0.0', 'ffff:ffff:ffff:ffff::'])
```

**Comment by TomAugspurger** [comment]
Closed by #30
